lib/xandra/cluster.ex

```diff
@@ -482,7 +504,7 @@ defmodule Xandra.Cluster do
     case Supervisor.start_child(pool_supervisor, child_spec) do
       {:ok, pool} ->
         _ = Logger.debug("Started connection to #{inspect(address)}")
-        %{state | pools: Map.put(pools, address, pool)}
+        %{state | pools: Map.put(pools, {address, data_center}, pool)}
```

Comment:
Question: is the address unique per data center, or unique across data centers? If the former, this is good, but if the latter, maybe we can store this as `address => {data_center, pool}`.

Reply:
Thanks. I'd say the address should be unique across data centers, I'll apply your suggestion :+1: